src/components/NcColorPicker/NcColorPicker.vue

```diff
@@ -229,18 +275,19 @@ const normalizedPalette = computed(() => {
 		}
 	}
 
-	return palette.map((item) => ({
+	return palette.map((item: Color | string) => ({
 		color: typeof item === 'object' ? item.color : item,
 		name: typeof item === 'object' && item.name
 			? item.name
-			: t('A color with a HEX value {hex}', { hex: item.color }),
+			: t('A color with a HEX value {hex}', { hex: typeof item === 'string' ? item : item.color }),
 	}))
 })
 
-const contrastColor = computed(() => {
-	const black = '#000000'
-	const white = '#FFFFFF'
-	return calculateLuma(currentColor.value) > 0.5 ? black : white
+const isCustomColor = computed(() => {
```

Comment:
There's no option to clear selection from the default pallete, probably because it doesn't treated as custom color?

<img width="235" height="153" alt="Image" src="https://github.com/user-attachments/assets/3f972d5e-45fd-4b1d-8b08-502de295fa50" />